Modules/SubModule_ASL/xASL_wrp_ResampleASL.m

```diff
@@ -222,7 +227,18 @@ function xASL_wrp_ResampleASL(x)
     elseif round(dim4/2)~=dim4/2

```

Comment:
Can this be the case for Hadamard or Walsh?

Reply:
So I guess @amahroo is checking for uneven numbers here 🤔

Reply:
I didn't make any changes in the resample function, no idea :/

Reply:
Ahhh true, no one changed the code here. Is this one of your "teaching" questions @HenkMutsaerts ?

Reply:
since we have multiples of 4 and 8, this should not be the case for Hadamard/Walsh